packages/main/src/SliderBase.hbs

```diff
@@ -1,10 +1,10 @@
 <div
 	class="ui5-slider-root"
-	@mousedown="{{_handleDown}}"
-	@touchstart="{{_handleDown}}"
-	@pointerdown="{{_handleDown}}"
-	@mouseover="{{_handleMouseOver}}"
-	@mouseout="{{_handleMouseOut}}"
+	@mousedown="{{_onmousedown}}"
+	@touchstart="{{_ontouchstart}}"
+	@pointerdown="{{_onpointerdown}}"

```

Comment:
According to mdn the pointerdown is not supported in Safari and Safari iOS, did you test it there
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/pointerdown_event

Reply:
Yes, it is working ok, it uses the mousedown handling